zmij.cc

```diff
@@ -783,22 +789,23 @@ inline auto is_big_endian() noexcept -> bool {
 }
 
 inline auto clz(uint64_t x) noexcept -> int {
-  assert(x != 0);
-#if ZMIJ_HAS_BUILTIN(__builtin_clzll)
```

Comment:
Why removing this branch?
Most x86 linux users with older (like <gcc 15) compilers with default args does not have `__builtin_ia32_lzcnt_u64` or `__builtin_clzg`.

Reply:
Ugh, I wasn't aware that the function is a new addition. I picked the GCC 9 manual at random and it indeed isn't there. Ok, will adapt. Thanks!